tensorboard/webapp/util/ui_selectors_test.ts

```diff
@@ -26,7 +26,7 @@ import {
 } from '../experiments/store/testing';
 import {
   buildFeatureFlag,
-  buildFeatureFlagState as buildFeatureFlagState,
```

Comment:
Admittedly not entirely certain why the formatter now catches this. Perhaps it has something to do with the TypeScript upgrade.